roles/common/tasks/main.yml

```diff
@@ -23,12 +23,11 @@
     retries: 5
   when: ansible_distribution == "Ubuntu"
 
-#FIXME
-- name: set selinux to permissive on redhat
+- name: set selinux to enforcig  on redhat
   selinux:
     policy: targeted
-    state: permissive
-  when: ansible_distribution == "RedHat"
+    state: "{{ (ansible_distribution == 'RedHat') | ternary('enforcing', 'permissive') }}"
+  when: ursula_os == "rhel"
```

Comment:
I think I'd like to see selinux state come from vars `common.selinux.enabled` and `common.selinux.mode: enforcing|permissive` or something like that, rather than forcing it based on OS.